packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.tsx

```diff
@@ -1,32 +1,33 @@
-import React, { CSSProperties, useCallback, useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import classNames from 'classnames';
 import { useCombobox } from 'downshift';
 import ArrowIcon from '../../../public/images/pointed-down-arrow-icon.svg';
 import reactNodeToString from '../../../utils/reactNodeToString';
 import { TGenericSizes } from '../types';
 import { WalletText } from '../WalletText';
+import { WalletTextField } from '../WalletTextField';
 import './WalletDropdown.scss';
 
 type TProps = {
```

Comment:
```suggestion
interface WalletDropdownProps extends WalletTextfieldProps = {
```

Reply:
with this you dont have to specify each and every prop that you share with WalletTextField

Reply:
Instead of extending the whole WalletTextfieldProp and having props that we don't need, maybe we could pick only the  needed props from it instead?